platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/LogoutRequestService.java

```diff
@@ -68,16 +72,30 @@
 public class LogoutRequestService {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(LogoutRequestService.class);
-
   private static final String SAML_REQUEST = "SAMLRequest";
-
   private static final String SAML_RESPONSE = "SAMLResponse";
-
   private static final String RELAY_STATE = "RelayState";
-
   private static final String SIG_ALG = "SigAlg";
-
   private static final String SIGNATURE = "Signature";
+  public static final String IDP_REALM_NAME = "idp";
+  public static final String NO_SUPPORT_FOR_POST_OR_REDIRECT_BINDINGS =
+      "The identity provider does not support either POST or Redirect bindings.";
+  public static final String ROOT_NODE_NAME = "root";
+  public static final String UNABLE_TO_CREATE_LOGOUT_REQUEST = "Failed to create logout request";
+  public static final String UNABLE_TO_CREATE_LOGOUT_RESPONSE = "Failed to create logout response";
+  public static final String UNABLE_TO_DECODE_AND_INFLATE_LOGOUT_REQUEST =
+      "Unable to decode and inflate logout request";
+  public static final String UNABLE_TO_DECODE_AND_INFLATE_LOGOUT_RESPONSE =
+      "Unable to decode and inflate logout response.";
+  public static final String UNABLE_TO_DECRYPT_LOGOUT_REQUEST =
+      "Failed to decrypt logout request params. Invalid number of params";
+  public static final String UNABLE_TO_PARSE_LOGOUT_REQUEST = "Unable to parse logout request";
+  public static final String UNABLE_TO_PARSE_LOGOUT_RESPONSE = "Unable to parse logout response";
+  public static final String UNABLE_TO_SIGN_LOGOUT_RESPONSE = "Failed to sign logout response";
+  public static final String UNABLE_TO_VALIDATE_LOGOUT_REQUEST =
+      "Unable to validate logout request";
+  public static final String UNABLE_TO_VALIDATE_LOGOUT_RESPONSE =
+      "Unable to validate logout response";
```

Comment:
I did some Sonar Lint cleanup on this class. Most of the changes were to convert strings into constants. This turned out to be important because the unit tests were testing for the exact strings of error messages generated by this class!

At some point I became concerned that refactoring was going to break something, so I stopped.